testproject/Assets/Tests/Runtime/NetworkSceneManagerTests.cs

```diff
@@ -43,7 +43,7 @@ private class SceneTestInfo
 
 
         [UnityTest]
-        public IEnumerator SceneLoadingAndNotifications()
+        public IEnumerator SceneLoadingAndNotifications([Values(LoadSceneMode.Single,LoadSceneMode.Additive)] LoadSceneMode clientSynchronizationMode)
```

Comment:
so how we verify whether single or additive modes result into single or additive on the client-side?
I'm not seeing any checks or asserts — does this mean that "works fine" implicitly means "unit-tested"?

Reply:
Good point, will add verification that the SceneEventData on the client side reflects the clientSynchronization mode set.